### PR TITLE
AP_Logger: add increment metadata to allow <1Hz logging

### PR DESCRIPTION
**Pull request by Hwurzburg**
NFC....without this user can't set this below 1Hz in normal GCS dialogs

### CODE REVIEW

**Comment by amilcarlucas** [comment]
Isn't the correct fix to declare a step size?

**Comment by Hwurzburg** [comment]
Param is float

File: libraries/AP_Logger/AP_Logger.cpp [hunk 1]
```diff
@@ -136,7 +136,7 @@ const AP_Param::GroupInfo AP_Logger::var_info[] = {
     // @DisplayName: Maximum logging rate for file backend
     // @Description: This sets the maximum rate that streaming log messages will be logged to the file backend. A value of zero means that rate limiting is disabled.
     // @Units: Hz
-    // @Range: 0 1000
+    // @Range: 0.1 1000
```
Comment (khancyr) [inline libraries/AP_Logger/AP_Logger.cpp:139]: 0.016 maybe so you can have a full min range?

**Review by khancyr — APPROVED** [review]

File: libraries/AP_Logger/AP_Logger.cpp (hunk 1 above)
Comment (Hwurzburg) [reply libraries/AP_Logger/AP_Logger.cpp:139]: I think logging once every ten seconds is slow enough...but am open to the group's view

**Comment by IamPete1** [comment]
0 is a valid value meaning disabled? What is the issue with GCS? I just tried mission planner full list, I can put in a decimal.

Maybe adding `// @Increment: 0.1` will fix?

**Comment by Hwurzburg** [comment]
yeah,  that would be better....will change to that

**Review by IamPete1 — APPROVED** [review]

**Review by tridge — APPROVED** [review]

**Pull request merged**